assets/src/components/parts/janus-text-flow/Markup.tsx

```diff
@@ -146,6 +146,13 @@ const Markup: React.FC<any> = ({
   // TODO: support MathJax
   // TODO: support templating in text
   // TODO: support tables, quotes, definition lists?? form elements???
+
+  // replace all recurring spaces with &nbsp; but not the first one
+  processedText = processedText.replace(
```

Comment:
removed `!processedText.trim()` condition, since it was used for strings only with recurring blank spaces (without any characters in it). Now it will consider, all strings with recurring blank spaces to replace two spaces "  " with " nbsp;"